### PR TITLE
Added in basic buttons for adding a review as well as seeing more

### PR DESCRIPTION
**Pull request by zman811**
I added in the buttons for adding and seeing more reviews, currently, they do nothing but I will add in the functionality when I add in what they go to

### CODE REVIEW

File: client/src/components/Reviews/components/ReviewList.jsx [hunk 1]
```diff
@@ -12,19 +12,39 @@ const List = styled.div`
...
+  background: none!important;
+  border: none;
+  padding: 0!important;
+  position: absolute;
+  right:  60%;
   bottom:  2%;
 `;
 
 function ReviewList() {
+  function AddReview(e) {
```
Comment (slargman) [inline client/src/components/Reviews/components/ReviewList.jsx:33]: does this not cause a naming collision with AddReview on line 15?

File: client/src/components/Reviews/components/ReviewList.jsx [hunk 2]
```diff
@@ -12,19 +12,39 @@ const List = styled.div`
...
+    e.preventDefault();
+    // TODO make a request to the api to get more/will change which reviews are being displayed
+  }
   return (
     <List>
       <div>there are some reviews that are not sorted</div>
       <Review />
       <Review />
-      <Buttons>Test</Buttons>
+      <AddReview onClick={AddReview}> Add review</AddReview>
```
Comment (slargman) [inline client/src/components/Reviews/components/ReviewList.jsx:46]: You're using AddReview to refer to two different things here. Doesn't this cause an error? I would suggest being more explicit with your naming, e.g. AddReviewButton and handleAddReviewClick. I try to prefix my event handlers with "handle..." in an attempt to be more explicit about what the purpose of the function is.

**Review by slargman — COMMENTED** [review]
I think you'd have to changed some of your variable names to avoid collisions. But if the code is running without errors maybe it's fine? I would still say it's not the best style though.

I'm looking into setting up a theme and a basic shared component library for styled components that we could draw from. This would avoid the need to duplicate some of these styling things for buttons, but we can refactor that if we get there.

File: client/src/components/Reviews/components/ReviewList.jsx (hunk 2 above)
Comment (zman811) [reply client/src/components/Reviews/components/ReviewList.jsx:46]: ah I didn't even notice that. yeah ill change that before I merge anything

**Review by slargman — APPROVED** [review]

**Pull request merged**